bin/git.sync.by-patches.sh

```
@@ -0,0 +1,837 @@
+source /dev/null
+# shellcheck disable=SC1090 source=../.scripts/_logger.sh
+source /dev/null
+# shellcheck disable=SC1090 source=../.scripts/_commons.sh
+source /dev/null
+# shellcheck disable=SC1090 source=../.scripts/_dependencies.sh
+source "${E_BASH}/_dependencies.sh"
+# shellcheck disable=SC1090 source=../.scripts/_tmux.sh
+source "${E_BASH}/_tmux.sh"
+
+# Initialize logger for this script
+logger:init sync "[${cl_blue}sync${cl_reset}] "
```

Shellcheck (0.10.0)

[warning] cl_blue is referenced but not assigned.

(SC2154)